main.go

```diff
@@ -11,30 +12,31 @@ import (
 	"github.com/araddon/dateparse"
 )
 
-func main() {
-
+func run() (exitStatus int, err error) {
 	fHeight := flag.Int64("height", 0, "specific height to estimate the time of")
 	fDate := flag.String("date", "", "specific time to estimate height at, e.g. Mon Jan 2 15:04:05 MST 2006")
-	fSamples := flag.Int64("samples", 100, "number of samples to take")
+	fSamples := flag.Int64("samples", 100, "count of samples to take")
```

Comment:
"number of samples" is idiomatic, IME. count is an observation, not an input.
(not critical)